waf_tools/magnum.py

```diff
@@ -214,9 +214,13 @@ def fatal(required, msg):
             conf.end_msg(opengl_include_dir)
 
             conf.start_msg('Magnum: Checking for OpenGL lib')
```

Comment:
Can you put this inside the if? No reason to say that we are looking for OpenGL if we are not.